library/std/src/error.rs

```diff
@@ -1618,11 +1610,11 @@ impl Report<Box<dyn Error>> {
     fn backtrace(&self) -> Option<&Backtrace> {
         // have to grab the backtrace on the first error directly since that error may not be
         // 'static
-        let backtrace = self.error.backtrace();
+        let backtrace = self.error.request_ref();
```

Comment:
I could have also written these as `.request_ref::<Backtrace>()` but its unambiguous here so I let it infer to show how it can be brief